tests/plugins/uploadwidget/additionalrequestparameters.js

```diff
@@ -112,10 +112,18 @@
 
 			File = window.File;
 
+			// Edge 18+ upstream issue breaks tests, workaround by forcing polyfill (#3184).
+			// developer.microsoft.com/en-us/microsoft-edge/platform/issues/22326784
+			if ( CKEDITOR.env.edge && CKEDITOR.env.version >= 18 ) {
+				assert.ignore();
+			}
+			var isEdge18 = CKEDITOR.env.edge && CKEDITOR.env.version >= 18;
```

Comment:
First you ignore the test then make Edge 18 detection to adjust test flow 🤔 Looks like one of these is a leftover?

Reply:
Yup, probably leftover, it passes on Edge 18 now, so no need to ignore it.